startup.sh

```diff
@@ -17,18 +17,19 @@ if [ $0 == "zsh" ]; then
     rc_file="$HOME/.zshrc"
 fi
 
+### Install software dependencies
 command_name=brew
 if ! type "$command_name" > /dev/null; then
-    echo "\n${purple}Installing Homebrew${normal}\n"
+    echo -e "\n${purple}Installing Homebrew${normal}\n"
```

Comment:
Add `-e` was necessary to actually print the newline rather than the characters.

<img width="423" alt="newlines" src="https://user-images.githubusercontent.com/507283/62839806-ffbbf800-bc5d-11e9-982c-184befdec918.png">